### PR TITLE
Setting 6 max versions to columns in Cluster Test.

### PR DESCRIPTION
**Pull request by sduskis**
This should fix the failures we've been seeing in Jenkins

### CODE REVIEW

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java [hunk 1]
```diff
@@ -216,10 +218,12 @@ private Cluster createACluster(BigtableClusterAdminClient client, String zoneNam
   }
 
   private void createTable(Admin admin, TableName tableName) throws IOException {
-    HTableDescriptor descriptor = new HTableDescriptor(tableName);
-    descriptor.addFamily(new HColumnDescriptor(COLUMN_FAMILY));
-    admin.createTable(descriptor);
-    Assert.assertTrue("Table does not exist", admin.tableExists(tableName));
+    HColumnDescriptor hcd = new HColumnDescriptor(COLUMN_FAMILY).setMaxVersions(MAX_VERSIONS);
```
Comment (mgarolera) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java]: family1?

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java [hunk 2]
```diff
@@ -63,9 +63,11 @@
  */
 public class TestClusterAPI {
 
-  private static final int MAX_WAIT_SECONDS = 20;
-  private static final String TEST_CLUSTER_ID = "test-cluster-api";
+  public static final int MAX_WAIT_SECONDS = 20;
+  public static final String TEST_CLUSTER_ID = "test-cluster-api";
   public static final byte[] COLUMN_FAMILY = Bytes.toBytes("test_family");
```
Comment (mgarolera) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java]: COLUMN_FAMILY1?

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java (hunk 1 above)
Comment (mgarolera) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java]: Why are two families needed? no test accesses COLUMN_FAMILY2

File: bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java (hunk 2 above)
Comment (sduskis) [inline bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java]: I copied the IntegrationTest's conventions.  If we change the Cluster Test names, we should change IntegrationTest names as well.  I'll make it so.

**Comment by sduskis** [comment]
I removed CLUSTER_FAMILY_2.  PTAL

**Comment by mgarolera** [comment]
LGTM

**Pull request merged**